code/modules/recycling/disposal/bin.dm

```diff
@@ -412,16 +412,16 @@
 	var/atom/L = loc //recharging from loc turf
 
 	var/datum/gas_mixture/env = L.return_air()
+	if(!env.temperature)
+		return
 	var/pressure_delta = (SEND_PRESSURE*1.01) - air_contents.return_pressure()
 
-	if(env.temperature > 0)
-		var/transfer_moles = 0.1 * pressure_delta*air_contents.volume/(env.temperature * R_IDEAL_GAS_EQUATION)
-
-		//Actually transfer the gas
-		var/datum/gas_mixture/removed = env.remove(transfer_moles)
-		air_contents.merge(removed)
-		air_update_turf()
+	var/transfer_moles = 0.1 * pressure_delta*air_contents.volume/(env.temperature * R_IDEAL_GAS_EQUATION)
```

Comment:
mind putting another bracket around these so it's a bit easier to read at a glance?

Reply:
sure I can do that

Reply:
wait around what? the multiply before the divisor?

Reply:
oh one for the top and one for the bottom, can do

Reply:
fixed